### PR TITLE
Humans_Engine: Add previous version attribute to CvalueAnalysis

### LINKED ISSUE #2391: Humans_Engine: Versioning needed for changes to CValueAnalysis
#### Broken rules:
as per title

### PR DESCRIPTION
**Pull request by rolyhudson**
### NOTE: Depends on 
https://github.com/BHoM/BHoM/issues/1221
   
### Issues addressed by this PR

Property name change versioned for CValueSettings
Previous version attribute for the CValueAnalysis method

Closes #2391 and https://github.com/BHoM/BHoM/issues/1221

### Test files
Old components and old json strings in this [file](https://burohappold.sharepoint.com/:u:/r/sites/BHoM/02_Current/12_Scripts/02_Pull%20Request/BHoM/BHoM/Humans_oM/%231221-CvalueVersioning/%231221_cvalueversioning.gh?csf=1&web=1&e=D0d4O0)

### CODE REVIEW

**Comment by rolyhudson** [comment]
@BHoMBot check-versioning

**Comment by rolyhudson** [comment]
@BHoMBot check-versioning

**Comment by rolyhudson** [comment]
@BHoMBot check versioning

File: Humans_Engine/Query/CvalueAnalysis.cs [hunk 1]
```diff
@@ -51,6 +52,7 @@ public static List<Cvalue> CvalueAnalysis(Audience audience, CvalueSettings sett
 
         /***************************************************/
 
+        [PreviousVersion("4.1", "BH.Engine.Humans.ViewQuality.Query.CvalueAnalysis(List<BH.oM.Humans.ViewQuality.Audience>, BH.oM.Humans.ViewQuality.CvalueSettings, BH.oM.Geometry.Polyline)")]
```
Comment (FraserGreenroyd) [inline Humans_Engine/Query/CvalueAnalysis.cs:55]:
```suggestion
        [PreviousVersion("4.1", "BH.Engine.Humans.ViewQuality.Query.CvalueAnalysis(Sytstem.Collections.Generic.List<BH.oM.Humans.ViewQuality.Audience>, BH.oM.Humans.ViewQuality.CvalueSettings, BH.oM.Geometry.Polyline)")]
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
This requested change should remove the final humans related versioning issue :smile:

File: Humans_Engine/Query/CvalueAnalysis.cs (hunk 1 above)
Comment (rolyhudson) [reply Humans_Engine/Query/CvalueAnalysis.cs:55]: Perfect thanks @FraserGreenroyd 👍

**Comment by rolyhudson** [comment]
@BHoMBot check versioning

**Comment by rolyhudson** [comment]
@BHoMBot check versioning

**Comment by rolyhudson** [comment]
@BHoMBot check versioning

**Review by FraserGreenroyd — DISMISSED** [review]
Versioning errors reduced, this LGTM, thanks @rolyhudson :smile:

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check serialisation

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core

**Comment by rolyhudson** [comment]
@FraserGreenroyd fixed a Query compliance issue. But I think branch name is missing a hyphen. I hope dispensation can be granted here and on the dependent PR?

**Comment by rolyhudson** [comment]
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check serialisation

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core

**Review by FraserGreenroyd — APPROVED** [review]

**Pull request merged**